action.yml

```diff
@@ -59,16 +59,23 @@ runs:
       run: |
         ${GITHUB_ACTION_PATH}/dependencies.R
         [ ! -f "./template.qmd" ] && cp ${GITHUB_ACTION_PATH}/template.qmd . || echo "./template.qmd already exists"
-        if [[ "$(echo -e "$(quarto --version)\n1.4.0" | sort -V | head -1)" == "1.4.0" ]]; then
-          echo "✅ Quarto CLI version sufficient (>= 1.4.0)"
+        MIN_QUARTO_VERSION="1.5.29"
+        if [[ "$(echo -e "$(quarto --version)\n${MIN_QUARTO_VERSION}" | sort -V | head -1)" == "${MIN_QUARTO_VERSION}" ]]; then
```

Comment:
So the newest Quarto release will be downloaded in all cases, except when Quarto 1.5.29 is found?

Reply:
Okay, but the suggested update will do nothing only if `quarto --version == ${MIN_QUARTO_VERSION}`. It will update Quarto if `quarto --version > ${MIN_QUARTO_VERSION}`.

Reply:
Okay, I just wanted to make sure we have a common understanding 😁